index.php.in

```diff
@@ -104,26 +142,58 @@ header("X-Powered-By: ");
       <article>
         <header><h2>Features</h2></header>
         <ul>
-          <li>You can create interpreter for your JSON-RPC service with <strong>one line of code</strong>.</li>
-          <li>Support for <strong>authentication</strong> (you can provide function when user enter login and password or if you use <strong>JSON-RPC</strong> it can <strong>automatically call login function</strong> on the server and pass token to all functions).</li>
-          <li>Stack of interpreters - you can create commands that trigger additional interpreters (eg. you can use couple of JSON-RPC service and run them when user type command).</li>
-          <li>Command Tree - you can use nested objects each command will invoke a function if the value is an object it will create new interpreter and use function from that object as commands. You can use as much nested commands as you like. if the value is a string it will create JSON-RPC service.</li>
+          <li>You can create an interpreter for your JSON-RPC service
+              with <strong>one line of code</strong>.</li>
```

Comment:
this is how the code should look like if you really want to have line wrapping in source code.